docs/running-headscale-container.md

```diff
@@ -44,7 +44,13 @@ curl https://raw.githubusercontent.com/juanfont/headscale/main/config-example.ya
 touch ./config/config.yaml
 ```
 
-Modify the config file to your preferences before launching Docker container.
+Modify the config file to your preferences before launching Docker container. Here are some settings that you likely want:
+```yaml
+server_url: http://your-host-name:8080 # Change to your hostname or host IP
```

Comment:
Can you put the comments above so the line does not get that long ?

Reply:
Fix in [73f1c06](https://github.com/juanfont/headscale/pull/521/commits/73f1c06f65e1b010da2bf37044d100f2f2af52c5)